modules/ROOT/pages/index.adoc

```diff
@@ -37,19 +37,19 @@ The Mule Healthcare Toolkit includes the following elements.
 
 == Installing the Mule Healthcare Toolkit
 
-The Mule Healthcare Toolkit is available for free on a trial basis. Contact mailto:info@mulesoft.com[MuleSoft Sales Inquiries] to request a license to use the Healthcare Toolkit components in a production environment.
+The Mule Healthcare Toolkit is available for free on a trial basis in non-production environments:
 
 . In Anypoint Studio, click the Exchange icon in the Studio taskbar.
 . Click Login in Anypoint Exchange.
 . Search for HL7 and click Install.
 . Follow the prompts to install the connector.
 . Repeat steps 3 and 4 to install the HL7 MLLP Connector too.
 
+To request a license to use the Healthcare Toolkit components in a production environment, contact your MuleSoft account representative or xref:https://www.mulesoft.com/contact[the MuleSoft sales team].
 
 == See Also
 
 * https://mulesoft.github.io/mule-hl7-connector/[HL7 Technical Reference]
 * http://www.hl7.org[Health Level Seven International]
 * Use the xref:connector-testpanel.adoc[Connector Testing with TestPanel] to test your HL7 connector application.
-* xref:mllp-connector.adoc[HL7 MLLP Reference]
-* mailto:info@mulesoft.com[MuleSoft Sales Inquiries]
\ No newline at end of file
+* xref:mllp-connector.adoc[HL7 MLLP Reference]
```

Comment:
I didn't repeat the link, there's no need.

Reply:
I have been bolding the UI controls, for example, Click *Login* etc, which brings me to my next comment--the dialog actually says Sign in, so it would be Click *Sign in*
Although, this depends on whether or not it's a new session or an expired session. bleh You can probably just leave it.

Reply:
I wrote around this, saying Log into, instead of quoting the UI labels since they differ. Also rewrote for Mule 4.